lib/tasks/ci/redmine.rake

```diff
@@ -0,0 +1,90 @@
+namespace :ci do
+  namespace :redmine do
+    desc 'Set up Redmine'
+    task :setup do
+      ENV['REDMINE_HOSTNAME'] ||= ask('Redmine URL: ')
+      ENV['REDMINE_HOSTNAME'] = nil if ENV['REDMINE_HOSTNAME'] == ''
+
+      ENV['REDMINE_API_KEY'] ||= ask('Redmine API Key: ') { |q| q.echo = '*' }
+      ENV['REDMINE_API_KEY'] = nil if ENV['REDMINE_API_KEY'] == ''
+    end
+  end
+
+  namespace :redmine do
+    desc 'Update Redmine tickets'
+    task update_tickets: ['ci:rugged:setup', 'ci:redmine:setup'] do
+      @api_key = ENV['REDMINE_API_KEY']
+      @hostname = ENV['REDMINE_HOSTNAME']
+      next if @api_key.nil? || @hostname.nil?
+
+      require 'json'
+      require 'net/http'
+
+      @http = Net::HTTP.new(@hostname, 443)
+      @http.use_ssl = true
+
+      @headers = {
+        'X-Redmine-API-Key' => @api_key,
+        'Content-Type'      => 'application/json'
+      }
+
+      CLOSE_REGEX =
```

Comment:
I'd be inclined to leave these as variables, as the magic of rake means this is defining `Object::CLOSE_REGEX`.